sites/all/themes/humanitarianresponse/assets/sass/partials/_panel.scss

```diff
@@ -109,6 +109,23 @@
   }
 }
 
+/* hid contact name font style */
+
+.field-type-hid-contact-reference {
+    display: block;
+
+      h4 {
```

Comment:
Now that I see this. I think you should not restyle a h4 > strong but change the html instead.